src/markdown-pages/docs/first-steps/getting-started.md

```diff
@@ -9,6 +9,8 @@ You can install Galasa either from a downloadable zip file (zipped distribution)
 
 The zipped distribution allows users who do not have access to Maven Central, Eclipse Marketplace and Docker Hub from their company network to use Galasa. Users without restricted internet access who want to use open-source Galasa, can install the Galasa Eclipse plug-in directly from the external update site.
 
+As IP addresses and passwords are stored in Galasa, hard drive encryption is recommended.
```

Comment:
Can we rephrase to - As IP addresses and ports of test machines are stored within the cps.properties file on a users system, we recommend that hard drive encryption is turned on in the operating system where possible

Reply:
updated thanks